modules/ROOT/edits/manage-apis-with-service-mesh.adoc

```diff
@@ -0,0 +1,43 @@
+= Managing APIs with Anypoint Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, manage APIs
+
+image::manage-service-mesh-breadcrumb.png[]
+
+After you configure Anypoint Service Mesh, you can manage both Mule and non-Mule applications from Anypoint API Manager, including applying policies to secure your microservices and viewing analytics data regarding the usage of your APIs. 
+
+
+== Apply Policies to Your APIs
+
+Not all policies available in Anypoint Platform are applicable to Anypoint Service Mesh. Only a subset of those policies apply. Additionally, the configurations within each of the policies that are applicable to Anypoint Service Mesh also differ from the policies that are applicable to Anypoint Platform.
+
+For example, you can apply the Rate Limiting policy to define the number of requests that you want your service to receive within a specified period of time.
```

Comment:
How is this an example of the configuration applicability difference: of the sentence directly above?

Reply:
Noted, will find out.